src/devtoolsPanel.ts

```diff
@@ -270,19 +285,26 @@ export class DevToolsPanel {
         context: vscode.ExtensionContext,
         telemetryReporter: Readonly<TelemetryReporter>,
         targetUrl: string,
-        config: IRuntimeConfig) {
+        config: IRuntimeConfig,
+        stringMap: string = "") {
         const column = vscode.ViewColumn.Beside;
 
         if (DevToolsPanel.instance) {
             DevToolsPanel.instance.panel.reveal(column);
         } else {
-            const panel = vscode.window.createWebviewPanel(SETTINGS_STORE_NAME, SETTINGS_WEBVIEW_NAME, column, {
+            let name = SETTINGS_WEBVIEW_NAME;
+            if (stringMap) {
+                const stringMapJSON = JSON.parse(stringMap);
+                name = stringMapJSON[name];

```

Comment:
Should we have a fallback in case the json is missing an entry for the name?

Reply:
it will fallback to the en-us version.